samples/snippets/quickstart.py

```diff
@@ -73,9 +73,14 @@ def list_all_instances(
         iterable collections of Instance objects as values.
     """
     instance_client = compute_v1.InstancesClient()
```

Comment:
The pagination example shown here in the quickstart is for `compute_v1.InstancesClient()`, while the Samples added below are for `compute_v1.ImagesClient()`, do you plan on providing samples for `InstancesClient()` as well? Users might get confused because it's not the exact same snippet of code they're trying to look more into.

Reply:
Not right now. The difference here is intentional.

ImagesClient.list() returns a simple list of results as [`ListPager` object ](https://googleapis.dev/python/compute/latest/compute_v1/images.html#google.cloud.compute_v1.services.images.ImagesClient.list)- which you iterate over like a list.

InstancesClient.aggregated_list() returns a dict-like result of [`AggregatedListPager` object](https://googleapis.dev/python/compute/latest/compute_v1/instances.html#google.cloud.compute_v1.services.instances.InstancesClient.aggregated_list).

We just try to show how different response types work. Examples on how to use various API services will be coming later.

Reply:
>Examples on how to use various API services will be coming later.

Sounds good!